index.html

```diff
@@ -31,6 +31,25 @@
       </div>
     </div>
   </nav>
+
+  <header class="bg__color--blue">
+    <div class="body__container body__container--header">
+      <div class="header__slogan">
+        <h1 class="header__slogan--text">We are creatives</h1>
+        <div class="header__slogan--arrow">
+          <a href="#articles__container">
```

Comment:
Jeśli ten `<a>` ma przenieść do sekcji poniżej, to jeszcze brakuje zdefiniowania **id="articles__container"** na np. elemencie `<div class="articles__container">`

Reply:
dobrze,  dodałam id="articles__container" do <div class="articles__container">